src/components/Publish/_utils.ts

```diff
@@ -485,7 +485,7 @@ export async function verifyRawServiceCredential(
   verified: boolean
   complianceApiVersion?: string
   idMatch?: boolean
-  matchVerifiable?: boolean
+  isIdMatchVerifiable?: boolean | string
```

Comment:
suggestion(blocking): given the recent refactorings it looks like it could make sense reworking this variable a bit. It no longer needs to serve as a boolean value, but rather is sort of a warning message that should be displayed in case some potential verification issues have been found.

We can simply use an optional `string` value here and evaluate based on its presence if a warning should be displayed to the user.